### PR TITLE
Adds darkmode properly this time :tm:

### PR DESCRIPTION
**Pull request by Kmc2000**
If we rebase to /tg/ I'll re-port this myself

:cl: Kmc2000
add: Darkmode
/:cl:

(Hyperlink colour is now fixed)

### CODE REVIEW

File: interface/skin.dmf [hunk 1]
```diff
@@ -65,26 +65,30 @@ window "mainwindow"
...
+		background-color = #272727
 		saved-params = "splitter"
 		left = "mapwindow"
 		right = "infowindow"
 		is-vert = true
 	elem "input"
 		type = INPUT
 		pos = 3,420
-		size = 517x20
+		size = 595x20
```
Comment (alexkar598) [inline interface/skin.dmf]: why change the size of this?

File: interface/skin.dmf [hunk 2]
```diff
@@ -163,61 +177,75 @@ window "infowindow"
...
+		background-color = #272727
 		saved-params = "splitter"
 		left = "statwindow"
 		right = "outputwindow"
 		is-vert = false
 	elem "changelog"
 		type = BUTTON
 		pos = 16,5
-		size = 104x20
+		size = 80x20
```
Comment (alexkar598) [inline interface/skin.dmf]: why?

File: interface/skin.dmf [hunk 3]
```diff
@@ -163,61 +177,75 @@ window "infowindow"
...
 		anchor2 = 19,0
+		text-color = #99aab5
+		background-color = #494949
 		saved-params = "is-checked"
 		text = "Changelog"
 		command = "changelog"
 	elem "rules"
 		type = BUTTON
-		pos = 120,5
-		size = 100x20
```
Comment (alexkar598) [inline interface/skin.dmf]: why did you change all the buttons size and position?

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by Kmc2000** [comment]
Adding a pref for it. DNM

**Pull request merged**